searchcore/src/vespa/searchcore/proton/server/prune_session_cache_job.cpp

```diff
@@ -19,7 +16,7 @@ PruneSessionCacheJob::PruneSessionCacheJob(ISessionCachePruner &pruner,
 bool
 PruneSessionCacheJob::run()
 {
-    TimeStamp now(ClockSystem::now());
+    fastos::TimeStamp now = fastos::ClockSteady::now();
```

Comment:
what kind of time stamps are put in the cache?
if for example request start time is used, this could lead to really long lived cache entries...

Reply:
Request startime is also a steady one now.